apps/portlet-documents/src/main/resources/locale/portlet/attachments_en.properties

```
@@ -13,6 +13,7 @@ attachments.drawer.maxFileSize=Maximum file size : {0} MB
 attachments.drawer.maxFileCountLeft=Max files left: {0}
 attachments.drawer.maxFileCount=Max files: {0}
 attachments.drawer.maxFileSize.error=You can\u2019t upload files larger than {0} MB
+attachments.drawer.nullFileSize.error=Attached file size must be greater than 0 bytes
```

You can't upload an empty file
```suggestion
attachments.drawer.nullFileSize.error=You can't attach an empty file
```